pxtarget.json

```diff
@@ -687,7 +687,7 @@
         "ocvFrameUrl": "https://admin-ignite.microsoft.com"
     },
     "queryVariants": {
-        "skillsMap=1": {
```

Comment:
Do we know if this could break partners who might embed us? (For example, not sure if code ninjas uses skill maps).

My guess is probably not, but we should make sure.

Reply:
Ah, good point, @kiki-lee, do you know if Code Ninjas or anyone else uses this flag?

Reply:
Or @Jaqster?

Reply:
They don't use our tutorials or skillmaps, so I assume it's fine. Also, they're pinned to an older version of Arcade.  I can't think of anyone else who serves us in a version that's not the offline app. I'm sure @abchatra would know better.

Reply:
I don't believe  embedding skillmap works today. This should be ok

Reply:
Also we don't have to fix this. This is not use facing string.

Reply:
If it should be fine, I'm not sure I see the problem in updating it if we want the correct spelling of skillmap to persist throughout the code base.